### PR TITLE
refactor: Refactored calculateMaxOfMin() (#5)

### PR DESCRIPTION
**Pull request by chubretovic**
Refactored the function calcultateMaxOfMin form the class MaximinumMinimumWindow.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +40,25 @@
...
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
-</project>
\ No newline at end of file
+</project>
```
Comment (siriak) [inline pom.xml:64]:
```suggestion
</project>

```

File: src/main/java/com/thealgorithms/datastructures/stacks/MaximumMinimumWindow.java [hunk 2]
```diff
@@ -100,7 +75,8 @@ public static void main(String args[]) {
         int[] arr = new int[]{10, 20, 30, 50, 10, 70, 30};
         int[] target = new int[]{70, 30, 20, 10, 10, 10, 10};
         int[] res = calculateMaxOfMin(arr, arr.length);
-        assert Arrays.equals(target, res);
+        //assert Arrays.equals(target, res);
```
Comment (siriak) [inline src/main/java/com/thealgorithms/datastructures/stacks/MaximumMinimumWindow.java:78]: Please remove main()

**Review by siriak — CHANGES_REQUESTED** [review]
Looks good, only a couple of comments

**Comment by siriak** [comment]
And please update to the latest master

**Comment by siddhant2002** [comment]
Can I work under this issue?

**Comment by siriak** [comment]
@siddhant2002 yes, you can

**Pull request closed without merging**